src/actuator/ActuatorParsing.C

```diff
@@ -209,6 +209,10 @@ epsilon_parsing(int iTurb, const YAML::Node& turbNode, ActuatorMeta& actMeta)
   } else if (epsilon_chord) {
     // require epsilon chord and epsilon min
     get_required(turbNode, "epsilon_chord", epsilonTemp);
+      if (
+        epsilonTemp[0] == epsilonTemp[1] && epsilonTemp[1] == epsilonTemp[2]) {
```

Comment:
Comparing doubles with `==` probably might not do what you want.

Reply:
@sayerhs are you referring to precision issues? I didn't think that would be an issue here since these are coming from the yaml node with zero computation.  Can the assignment of the exact same number still lead to different double precision values?